packages/build/tests/core/tests.js

```diff
@@ -533,6 +533,39 @@ test.serial('configFileDirectories is passed to zip-it-and-ship-it', async (t) =
   ])
 })
 
+test('internalSrcFolder is passed to zip-it-and-ship-it', async (t) => {
+  const zipItAndShipItSpy = sinon.spy(zipItAndShipIt, 'zipFunctions')
+
+  await new Fixture('./fixtures/functions_internal_src_folder').withFlags({ mode: 'buildbot' }).runWithBuild()
+  zipItAndShipItSpy.restore()
+  const { args: call1Args } = zipItAndShipItSpy.getCall(0)
+
+  const { internalSrcFolder, manifest } = call1Args[2]
+  const { functions } = await importJsonFile(manifest)
+
+  t.deepEqual(internalSrcFolder, join(FIXTURES_DIR, 'functions_internal_src_folder/.netlify/functions-internal'))
```

Comment:
You don't need `deepEqual` since you're just comparing strings.

```suggestion
  t.is(internalSrcFolder, join(FIXTURES_DIR, 'functions_internal_src_folder/.netlify/functions-internal'))
```